src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -23,6 +23,8 @@
     protected static final String EMPTY_JSON = "empty.json";
     protected static final String INVALID_JSON = "invalid.json";
     protected static final String INVALID_REPORT_JSON = "invalid-report.json";
+    
+    protected static final String SAMPLE_HOOKS_JSON = "sample-hooks.json";
```

Comment:
I don't want to extra test file. Instead:
- add single output and array aoutput into existing sample.json file
- add integration test that validates if the output is generated properly